config/katello.migrations/161004173201_foreman_ansible.rb

```diff
@@ -0,0 +1,5 @@
+answers['foreman::plugin::ansible'] ||= false
+scenario[:mapping]['foreman::plugin::ansible'] ||= {:dir_name => 'foreman', :manifest_name => 'plugin/ansible'}
```

Comment:
Not sure if it was this before, but this is a string and the foreman_proxy is a symbol. Is that a difference?